includes/sanitizers/class-amp-core-theme-sanitizer.php

```diff
@@ -1030,6 +1030,12 @@ static function() use ( $args ) {
 						}
 					}
 
+					@media (max-width: 999px) {
+						.show-modal {
+							display: unset !important;
```

Comment:
I'd like to avoid the use of `!important` if at all possible.

Can you try with the following, more specific selector: `amp-lightbox.cover-modal.show-modal` ? I think that should fix the problem as well without using `!important`.

Reply:
Sure that works and would prevent future CSS rule conflicts.